Modelica/Blocks/package.mo

```diff
@@ -1088,24 +1079,7 @@ the \"controlBus\". When typing \"realSignal1\" as \"New name\", a connection of
 <p>
 is generated and the \"controlBus\" contains the new signal \"realSignal1\". Modelica tools
 may give more support in order to list potential signals for a connection.
-For example, in Dymola all variables are listed in the menu that are contained in
-connectors which are derived by inheritance from \"controlBus\". Therefore, in
-<a href=\"modelica://Modelica.Blocks.Examples.BusUsage_Utilities.Interfaces\">BusUsage_Utilities.Interfaces</a>
-the expected implementation of the \"ControlBus\" and of the \"SubControlBus\" are given.
-For example \"Internal.ControlBus\" is defined as:
-</p>
-
-<pre>  <strong>expandable connector</strong> StandardControlBus
-    <strong>extends</strong> BusUsage_Utilities.Interfaces.ControlBus;
-
-    <strong>import</strong> SI = Modelica.SIunits;
-    SI.AngularVelocity    realSignal1   \"First Real signal\";
-    SI.Velocity           realSignal2   \"Second Real signal\";
-    Integer               integerSignal \"Integer signal\";
-    Boolean               booleanSignal \"Boolean signal\";
-    StandardSubControlBus subControlBus \"Combined signal\";
-  <strong>end</strong> StandardControlBus;
-</pre>
+</p>
```

Comment:
I'd prefer to keep only delete the first sentence of that paragraph.

Reply:
It's a trade-off:
The entire Modelica.Blocks.Examples.BusUsage will be less clear without the screenshots and documentation about connecting to parts. (The screenshots are less cluttered in current Dymola-version.) 
One possibility is to replace 
  "For example, if "sine.y" is connected to the "controlBus", the following menu pops-up in Dymola:" 
by 
  "For example, if "sine.y" is connected to the "controlBus", a tool may introduce a popup:"
Another is to view bus-connections as part of tool documentation.

Reply:
After some thought I believe 
 "For example, if "sine.y" is connected to the "controlBus", a tool may show a popup:"
is preferable - or perhaps:
  "For example, if "sine.y" is connected to the "controlBus", a popup may appear:"
(Possibly with some text saying that it doesn't look exactly like that.)

I don't think that a user already familiar with how buses work in their GUI will be confused by seeing something that is not exactly the same; hopefully their GUI will be more intuitive.

If they are not familiar with how buses work in their GUI it seems they will be more likely to use buses after reading the documentation with the screenshot.